projects/storefrontstyles/scss/layout/page-templates/_multistep-checkout-page.scss

```diff
@@ -5,22 +5,35 @@
   padding: 2rem 1.5rem 0 1.5rem;
   margin: auto;
   display: flex;
+  flex-direction: row;
+  flex-wrap: wrap;
+  justify-content: center;
+  align-content: flex-start;
 
   @include media-breakpoint-down(md) {
     display: inline-block;
   }
+
   cx-page-slot {
-    &.BodyContent,
-    &.SideContent {
-      display: block;
+    &.TopContent,
+    &.BottomContent {
+      --cx-flex: 0 1 100%;
     }
 
     &.BodyContent {
-      --cx-flex-basis: 200%;
+      --cx-flex: 1 1 auto;
+      --cx-flex-basis: 66%;
     }
 
     &.SideContent {
-      --cx-flex-basis: 100%;
+      --cx-flex: 1 1 auto;
+      --cx-flex-basis: 33%;
+    }
+
+    &.TopContent {
```

Comment:
Shouldn't we change --cx-flex-basis for BodyContent and SideContent instead of TopContent?